gensim/models/fasttext_inner.pyx

```diff
@@ -0,0 +1,544 @@
+#!/usr/bin/env cython
+# cython: boundscheck=False
+# cython: wraparound=False
+# cython: cdivision=True
+# coding: utf-8
+
+import cython
+import numpy as np
```

Comment:
I'm new to Cython, but isn't this redundant given the `cimport` below?

Reply:
The `import` gives access to numpy's Python functions while the `cimport` allows us to use it's C-modules.